tests/ImageSharp.Benchmarks/project.json

```diff
@@ -46,10 +45,23 @@
   "frameworks": {
     "net46": {
       "dependencies": {
+        "BenchmarkDotNet.Diagnostics.Windows": "0.10.1"
       },
       "frameworkAssemblies": {
         "System.Drawing": ""
       }
+    },
+    "netcoreapp1.1": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.1.0-*"
+        },
+        "BenchmarkDotNet": "0.10.2",
+        "CoreCompat.System.Drawing": "1.0.0-beta006",
+        "runtime.linux-x64.CoreCompat.System.Drawing": "1.0.0-beta009",
```

Comment:
After a little research it seems that we can safely reference all platforms it shall run on. It will simply choose the right native libs at runtime to bind to.